src/adservice/src/main/java/oteldemo/AdService.java

```diff
@@ -151,6 +155,15 @@ public void getAds(AdRequest req, StreamObserver<AdResponse> responseObserver) {
         AdRequestType adRequestType;
         AdResponseType adResponseType;
 
+        Baggage baggage = Baggage.fromContextOrNull(Context.current());
+        if (baggage != null) {
+          final String sessionId = baggage.getEntryValue("app.session.id");
+          span.setAttribute("app.session.id", sessionId);
```

Comment:
This should be using "session.id" as that is now the [semantic standard](https://opentelemetry.io/docs/specs/semconv/general/session/#attributes). We will need to update it in the frontend service as well.